src/utilities/widgets.py

```diff
@@ -1,11 +1,15 @@
 import copy
 import six
+import html
 import operator
 from functools import reduce
 
 from django.contrib.contenttypes.models import ContentType
 from django.db.models import Q
 
+import bleach
+from bleach.css_sanitizer import CSSSanitizer
+from django_summernote.widgets import SummernoteWidget, SummernoteInplaceWidget as SummernoteWidgetBase  # noqa
```

Comment:
I'd prefer not to rename Classes unless it's a common standard.  It makes code harder to read.  Let's just stick with `SummernoteInplaceWidget`  Because developers will have to dig through less code to understand what it means.